cc/position.h

```diff
@@ -30,6 +30,10 @@
 #include "cc/stone.h"
 
 namespace minigo {
+ namespace position {
```

Comment:
Can you remove this namespace please?
I'm actually surprised the test compiles, since it doesn't look like the function definition in the .cc file is in the same namespace. Did you run it with `--define=board_size=9`?

Reply:
the cc file defines in in minigo::position namespace.
I was trying to keep it out of the global minigo namespace, is this not the right choice?

Reply:
We don't really use inner namespaces on minigo, the project really isn't big enough to warrant them. I think the only one we have is minigo::sgf and that's because it felt weird to inject a name like Ast into the global namespace. In retrospect I should have just called the class SgfAst for consistency and not added the sgf inner namespace.